pages/_app.tsx

```diff
@@ -1,12 +1,12 @@
 import 'styles/globals.css'
 import Head from 'next/head'
 import type { AppProps } from 'next/app'
-import { LoginContext } from 'contexts/LoginContext'
-import useLogin from 'hooks/useLogin'
 import ResponsiveToolkit from 'components/ResponsiveToolkit'
+import { useLogin } from 'hooks/useLogin'
 
-function MyApp({ Component, pageProps }: AppProps) {
-  const $loginValue = useLogin({ timeout: 1500 })
+function MyApp({ Component: Page, pageProps }: AppProps) {
+  const $login = useLogin()
```

Comment:
This loading timeout is okay to disable now? Seems like disabling it would fix the case where you'd skip the loading screen after the timeout and see the page in an unauthed state, but potentially lock the site behind the loading screen if the API goes down

Reply:
That was my concern; I guess we have to weigh the difference between "it looks ok but maybe doesn't work" and "it never loads so I can't use it (and maybe it's also broken)". I tentatively think the latter is preferable but not certain on that.

Reply:
I chose the latter for now, we can revisit if it's untenable.